Keas.Mvc/Controllers/GroupController.cs

```diff
@@ -31,8 +33,33 @@ public async Task<IActionResult> Index(int id)
                 ErrorMessage = "Group not found or no access to Group";
                 return RedirectToAction("NoAccess", "Home");
             }
+            
+            var allTeams = await _context.TeamPermissions.Include(a => a.Team).Include(a => a.Role).Include(a => a.User).Where(a => a.Role != null && a.Role.Name == "DepartmentalAdmin").ToListAsync();

```

Comment:
maybe something like allTeamPermissions since it's not really allTeams. had me confused the first time i read through it